src/realm/sync/network/default_syncsocket.cpp

```diff
@@ -1,8 +1,8 @@
-#include <realm/util/ez_websocket.hpp>
+#include <realm/sync/network/default_syncsocket.hpp>
 
-#include <realm/util/websocket.hpp>
-#include <realm/util/network.hpp>
-#include <realm/util/network_ssl.hpp>
+#include <realm/sync/network/network.hpp>
+#include <realm/sync/network/network_ssl.hpp>
+#include <realm/sync/network/websocket.hpp>
 
 namespace realm::util::websocket {
```

Comment:
shouldn't the namespace reflect the new path? If that is the case, I think `sync` should be dropped from the file name.

Reply:
I wanted to do the namespace updates in a separate PR since that will be a big set of changes.
I was planning to update them like this:
* `realm::util::websocket` -> `realm::sync::websocket`
* `realm::util::network` -> `realm::sync::network`

Reply:
maybe you can consider the file renaming too

Reply:
Sorry, I don't follow - what are you recommending for the filenames?

Reply:
Updated filename `default_syncsocket.?pp` -> `default_socket.?pp`